src/app/modules/project/add/manual/manual.component.ts

```diff
@@ -61,6 +63,13 @@ export class ManualComponent implements OnInit {
    */
   public modulesConfigration = QuillUtils.getDefaultModulesConfiguration();
 
+  /**
+   * Configuration for file-picker
+   */
+  public acceptedTypes: Array<string> = [ 'image/png', 'image/jpg', 'image/jpeg' ];
+  public acceptMultiple: Boolean = false;
```

Comment:
You can omit the types when they can be automatically determined from the initialization. See: https://github.com/DigitalExcellence/dex-frontend/wiki/Coding-Guidelines#-type-definition